integration_tests/.scripts/integration_test_w_custom_module.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Expected input:
+# -d (database) target database for dbt
+
+while getopts 'd:' opt
+do
+  case $opt in
+    d) DATABASE=$OPTARG
+  esac
+done
+
+echo "Snowplow web integration tests: Seeding data"
+
+eval "dbt seed --target $DATABASE --full-refresh" || exit 1;
+
+echo "Snowplow web integration tests: Run 1: standard modules"
+
+eval "dbt run --target $DATABASE --full-refresh --vars 'teardown_all: true'" || exit 1;
+
+echo "Snowplow web integration tests: Run 2: standard modules"
+
+eval "dbt run --target $DATABASE" || exit 1;
+
+echo "Snowplow web integration tests: Run 3: Partial backfill of custom module + standard modules"
+# This tests the functionality of the snowplow_utils.is_run_with_new_events() macro
```

Comment:
Ideally, we assume that this macro does its job correctly, but have a thorough test for it in the snowplow-dbt-utils repo.

Reply:
Fair point. I am building out a series of tests for the macros in the utils package atm so should be able to cover this off in the next release.